q2_longitudinal/tests/test_longitudinal.py

```diff
@@ -115,6 +116,26 @@ def test_per_method_pairwise_stats_paired_nonparametric(self):
         res = _per_method_pairwise_stats(groups, paired=True, parametric=False)
         self.assertAlmostEqual(res['FDR P-value'][0], 0.0021830447373622506)
 
+    def test_calculate_variability(self):
+        res = _calculate_variability(md, 'Value')
+        for obs, exp in zip(res, [0.1808333, 0.0634548, 0.3711978, -0.0095311,
+                                  0.30774299, 0.05392367]):
+            self.assertAlmostEqual(obs, exp)
+
+    def test_compare_variances_equal_variance(self):
+        for method, exp in zip(['fligner', 'levene', 'bartlett'], [1, 1, 1]):
+            groups = [[1, 2, 3], [4, 5, 6]]
+            s, p, c = compare_variances(*groups, method=method)
+            self.assertAlmostEqual(p, exp)
+            self.assertEqual(c, 6)
+
+    def test_compare_variances_unequal_variance(self):
+        for method, exp in zip(['fligner', 'levene', 'bartlett'],
+                               [0.2984133, 0.3024691, 0.00131798]):
+            groups = [[1, 2, 3], [4, 15, 96]]
+            s, p, c = compare_variances(*groups, method=method)
+            self.assertAlmostEqual(p, exp)
+
```

Comment:
Can you add tests that check basic input/output testing of:

- `_add_sample_size_to_xtick_labels`
- `_control_chart_subplots`
- `_calculate_variability`
- `compare_variances`
- `_per_group_variance_comparison`
- `_multiple_tests_correction`
- `_control_chart`

Can you make sure that your tests reach into all of the branches in those functions? Thanks!